### ISSUE TITLE
Invalid sarif file produced by gosec

### ISSUE THREAD

**Issue by davidaparicio** [securego/gosec#877]
As https://github.com/securego/gosec/issues/870, but for the G114
Do we need to add G114 on cwe/data.go like this [commit](https://github.com/securego/gosec/commit/9aba3be19413a5670d8ee3584f4f2d0af17135b6) or it's a release issue ?

### Summary 

sarif file is invalid because of null object in relationships section of the second rule. Removing the relationship section entirely produces a valid sarif file.

```
Run github/codeql-action/upload-sarif@v2
Uploading results
Error details: instance.runs[0].tool.driver.rules[0].relationships[0] is not of a type(s) object
Error: Unable to upload "results.sarif" as it is not valid SARIF:
- instance.runs[0].tool.driver.rules[0].relationships[0] is not of a type(s) object
Error: Unable to upload "results.sarif" as it is not valid SARIF:
- instance.runs[0].tool.driver.rules[0].relationships[0] is not of a type(s) object
    at validateSarifFileSchema (/home/runner/work/_actions/github/codeql-action/v2/lib/upload-lib.js:203:15)
    at uploadFiles (/home/runner/work/_actions/github/codeql-action/v2/lib/upload-lib.js:267:9)
    at Object.uploadFromActions (/home/runner/work/_actions/github/codeql-action/v2/lib/upload-lib.js:139:18)
    at async run (/home/runner/work/_actions/github/codeql-action/v2/lib/upload-sarif-action.js:49:30)
    at async runWrapper (/home/runner/work/_actions/github/codeql-action/v2/lib/upload-sarif-action.js:71:9)
 ```

### Steps to reproduce the behavior

Code for reproducing this is available here: https://github.com/davidaparicio/namecheck/
Running the github workflow produces the error with the actions:

```
- name: Run Gosec Security Scanner
  uses: securego/gosec@master
  with:
    # we let the report trigger content trigger a failure using the GitHub Security features.
    args: '-no-fail -fmt sarif -out results.sarif ./...'

- name: Upload SARIF file
  uses: github/codeql-action/upload-sarif@v2
  with:
    # Path to SARIF file relative to the root of the repository
    sarif_file: results.sarif
```

### gosec version

securego/gosec@master

### Go version (output of 'go version')

embedded inside the Github Actions

### Operating system / Environment

Github Actions

### Expected behavior

Produces a valid sarif file

### Actual behavior

Produces a sarif file with a null relationship under rules.
See [workflow](https://github.com/davidaparicio/namecheck/actions/runs/3215413155/jobs/5256492632)

Locally, without sarif option

```
❯ gosec ./...

[/Users/daparici/code/github.com/davidaparicio/namecheck/cmd/server/main.go:52] - G114 (CWE): Use of net/http serve function that has no support for setting timeouts (Confidence: HIGH, Severity: MEDIUM)
    51:
  > 52: 	if err := http.ListenAndServe(":8080", nil); err != nil {
    53: 		log.Fatal(err)

Summary:
  Gosec  : dev
  Files  : 7
  Lines  : 574
  Nosec  : 0
  Issues : 1
  ```

**Comment by ccojocar** [comment]
The G114 is mapped to CWE-676 (see https://github.com/securego/gosec/blob/84661730b026a1dfae4b137b993f71b26fdda8cf/issue.go#L69) which was fixed by #870 but there isn't a release yet. The action is using the last released version of the container image. This should be fixed by the next release.

**Comment by davidaparicio** [comment]
Okay, thank for your explanation (I discover the mapping with the file ```issue.go```)
and also for your extreme rapidity to answer ! 😊
